sentry_sdk/integrations/strawberry.py

```diff
@@ -265,39 +265,27 @@ def _patch_execute():
     old_execute_async = strawberry_schema.execute
     old_execute_sync = strawberry_schema.execute_sync
 
+    @ensure_integration_enabled_async(StrawberryIntegration, old_execute_async)
     async def _sentry_patched_execute_async(*args, **kwargs):
         # type: (Any, Any) -> ExecutionResult
-        hub = Hub.current
-        integration = hub.get_integration(StrawberryIntegration)
-        if integration is None:
-            return await old_execute_async(*args, **kwargs)
-
         result = await old_execute_async(*args, **kwargs)
 
         if "execution_context" in kwargs and result.errors:
-            with hub.configure_scope() as scope:
-                event_processor = _make_request_event_processor(
-                    kwargs["execution_context"]
-                )
-                scope.add_event_processor(event_processor)
+            scope = Scope.get_isolation_scope()
```

Comment:
```suggestion
            scope = Scope.get_isolation_scope()
            scope.generate_propagation_context()
```
This is missing, right?